docs/topics/maven-multi-module.adoc

```diff
@@ -0,0 +1,51 @@
+= Run the {MavenNameTitle} with Multiple Modules
+
+To use the {MavenName} in a project with multiple modules, place the configuration inside the parent's `pom.xml`. During execution the {MavenName} will generate a single report that contains the analysis for the parent and any child modules.
+
+NOTE: It is strongly recommended to set `inherited` to false in multi-module projects; otherwise, the {MavenName} will be executed when each child is compiled, resulting in drastically increased run times.
```

Comment:
I might say that it also runs it multiple times when the parent module is compiled.